### PR TITLE
docs: add code of conduct

### PR DESCRIPTION
**Pull request by tuunit**
## Description
An official code of conduct for the project. Adopted from https://www.contributor-covenant.org/

## Motivation and Context
We should follow open source best practices and therefore I would like to add an official code of conduct to the repository.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (kvanzuijlen) [inline CODE_OF_CONDUCT.md:63]: This needs to be added; probably Slack?

**Review by kvanzuijlen — COMMENTED** [review]
I think this is a good addition, thx!

**Review by JoelSpeed — COMMENTED** [review]
I'm good with this, but we should fix the contact link

**Comment by tuunit** [comment]
@JoelSpeed I've add the slack link and mentioned the MAINTAINERS file.

**Review by JoelSpeed — APPROVED** [review]

**Pull request merged**